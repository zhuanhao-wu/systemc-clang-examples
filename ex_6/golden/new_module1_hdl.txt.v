module producer_0(
input wire [0:0] clock,
input wire integer in1,
output reg integer out,
output reg [0:0] ready 
);



endmodule // producer_0