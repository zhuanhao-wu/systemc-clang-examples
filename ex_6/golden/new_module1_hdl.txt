  [
    (hModule producer_0)
    [
      (hPortin clock)
      [
        (hType sc_in)
        (hType _Bool)
      ]
      (hPortin in1)
      [
        (hType sc_in)
        (hType int)
      ]
      (hPortout out)
      [
        (hType sc_out)
        (hType int)
      ]
      (hPortout ready)
      [
        (hType sc_out)
        (hType _Bool)
      ]
    ]
    [
    ]
  ]
